packages/openid4vc-holder/src/presentations/OpenId4VpHolderService.ts

```diff
@@ -0,0 +1,236 @@
+import type { PresentationSubmission } from './selection'
+import type { CredentialsForInputDescriptor } from './selection/types'
+import type { AgentContext, W3cCredentialRecord, W3cVerifiablePresentation } from '@aries-framework/core'
+import type {
+  DIDDocument,
+  PresentationDefinitionWithLocation,
+  SigningAlgo,
+  URI,
+  Verification,
+  VerifiedAuthorizationRequest,
+} from '@sphereon/did-auth-siop'
+import type { PresentationDefinitionV1 } from '@sphereon/pex-models'
+import type { W3CVerifiablePresentation } from '@sphereon/ssi-types'
+
+import {
+  AriesFrameworkError,
+  Buffer,
+  DidsApi,
+  getJwkClassFromKeyType,
+  getKeyFromVerificationMethod,
+  injectable,
+  TypedArrayEncoder,
+  W3cJsonLdVerifiablePresentation,
+  asArray,
+} from '@aries-framework/core'
+import { CheckLinkedDomain, OP, ResponseMode, SupportedVersion, VerificationMode } from '@sphereon/did-auth-siop'
```

Comment:
```suggestion
import { CheckLinkedDomain, OP as OpenIdProvider, ResponseMode, SupportedVersion, VerificationMode } from '@sphereon/did-auth-siop'
```

That is what it means, correct? OP seems quite vague.